marshmallow/schema.py

```diff
@@ -583,13 +586,17 @@ def _do_load(self, data, many=None, partial=None, postprocess=True):
         :param data: The data to deserialize.
         :param bool many: Whether to deserialize `data` as a collection. If `None`, the
             value for `self.many` is used.
-        :param bool partial: Whether to ignore missing fields. If `None`, the
-            value for `self.partial` is used.
+        :param bool|tuple partial: Whether to ignore missing fields. If `None`,
+            the value for `self.partial` is used. If its value is an iterable,
+            only missing fields listed in that iterable will be ignored.
         :param bool postprocess: Whether to run post_load methods..
         :return: A tuple of the form (`data`, `errors`)
         """
         many = self.many if many is None else bool(many)
-        partial = self.partial if partial is None else bool(partial)
+        if partial is None:
+            partial = self.partial
+        elif not utils.is_collection(partial):
```

Comment:
https://github.com/marshmallow-code/marshmallow/blob/bad7e56cd5602daf4eea31ce3cb49f07a5d7a9ba/marshmallow/marshalling.py#L266

That's the only place where partial is referenced in Unmarshaller and it's a simple `if partial` check. Changing the default of partial to `()` -- an empty tuple -- allows doing `if field_name in partial` there instead.

I misunderstood `partial = None` before, it's a check in the schema to decide which partial value to use. Instead, if we're using a tuple to declare specific fields to partial, why not do `partial = partial or self.partial`.